OpenXMLSDK.Engine/ReportEngine/DataContext/Charts/AxisModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OpenXMLSDK.Engine.ReportEngine.DataContext.Charts
+{
+    /// <summary>
+    /// Define Axis' details
+    /// </summary>
+    public class AxisModel
+    {
+        /// <summary>
+        /// Title
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// Title color
```

Comment:
Could you add a sample value to detail if we must provide the hexa code with or without # char